corehq/apps/export/models/new.py

```diff
@@ -1764,6 +1766,7 @@ def generate_schema_from_builds(
         force_rebuild=False,
         only_process_current_builds=False,
         task=None,
+        for_new_export_instance=False,
```

Comment:
Should this param also be added to the `generate_schema_from_builds()` function in the `SMSExportDataSchema` class? I'm not entirely sure if this class also gets used in the export view.

Reply:
Absolutely, good catch.
https://github.com/dimagi/commcare-hq/pull/34502/commits/902a3de50f0538a8a5d84c9e11f632b1b6711a9f

Reply:
Cool, we might also have to include the variable [here](https://github.com/dimagi/commcare-hq/blob/a96f2094c8ffc5f2258eadfa05c133f79d5461ba/corehq/apps/export/views/new.py#L295) so that new bulk case exports also receive the same Data Dictionary ordering.

Reply:
I have actually left bulk export out of scope for this PR.
Two reasons:
1. When we do a bulk export, the user is not shown the columns for the export on the UI.
2. For a bulk export, it seems, we [don't create a schema](https://github.com/dimagi/commcare-hq/blob/6b46f48241b95b8a7aff7a42ef467bdb8d946497/corehq/apps/export/views/new.py#L434) only.

I am thinking for bulk export, the order of case properties will not be relevant since there will be so many.
I am happy to add a ticket if you think we should address this.

Reply:
Thanks for sharing your reasoning. I can definitely see it being an okay outcome that users can't set the ordering for bulk case exports (since they can't access the ordering in the UI). Still, I think a ticket could still be a good idea just so that the team can have a chance to discuss this and determine whether there is any value in adding this for bulk case exports.

Reply:
Cool, I will add a ticket.